CRM/Contact/BAO/Query.php

```diff
@@ -1396,7 +1396,7 @@ public function query($count = FALSE, $sortByChar = FALSE, $groupContacts = FALS
       }
     }
     elseif ($sortByChar) {
-      $select = 'SELECT DISTINCT UPPER(LEFT(contact_a.sort_name, 1)) as sort_name';
+      $select = 'SELECT DISTINCT LEFT(contact_a.sort_name, 1) as sort_name';
```

Comment:
this strtoupper change seems to be a line here & one at the end - let's pul that out & test it separately

Reply:
+ the change in CRM/Utils/PagerAToZ.php (we are moving the UPPER() out of the SQL domain into the PHP domain).